packages/schematics/angular/secondary-entrypoint/index_spec.ts

```diff
@@ -0,0 +1,164 @@
+/**
+ * @license
```

Comment:
Can you please update https://github.com/angular/angular-cli/blob/main/tests/legacy-cli/e2e/tests/build/library-with-demo-app.ts to use the new schematic? so that we actually have an e2e that tests the generated code.